### PR TITLE
Release notes 3 3

### PR DESCRIPTION
**Pull request by karl-cardenas-coding**
## Describe the Change

This PR contains the release notes for 3.3

## Review Changes

💻 [Add Preview URL]()

### CODE REVIEW

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1185#" title="Author self-approved">karl-cardenas-coding</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/release-3-3/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: content/docs/00-release-notes.md [hunk 1]
```diff
@@ -13,6 +13,39 @@ import InfoBox from 'shared/components/InfoBox';
...
+
+* Users can now download all the clusters listed when applying a filter to the clusters list. 
+
+## Edge
+
+### Features:
+
+* Edge now supports the ability to load images from an external OCI registry.
+* The Edge Installer can now include preloaded content bundles containing packages and artifacts. This is useful for scenarios where you work with limited internet bandwidth or want to optimize the installation process.
+* Users can create a custom Edge Installer that contains content bundles, operating systems, and user-data.
```
Comment (ritawatson) [inline content/docs/00-release-notes.md:32]:
```suggestion
* Users can create a custom Edge Installer that contains content bundles, operating systems, and user data.
```

File: content/docs/00-release-notes.md [hunk 2]
```diff
@@ -13,6 +13,39 @@ import InfoBox from 'shared/components/InfoBox';
...
+* Support for creating an Edge Installer ISO based on Virtual Machine Disks (VMDK) is now available.
+* Support for generating random UUID values for the Edge host is now available. This addresses the issue of some devices resulting in the same UUID due to identical device identifiers.
```
Comment (ritawatson) [inline content/docs/00-release-notes.md:34]:
```suggestion
* Support for generating random UUID values for the Edge host is now available. This addresses the issue of some devices resulting in the same Universal Unique Identifier (UUID) due to identical device identifiers.
```

File: content/docs/00-release-notes.md [hunk 3]
```diff
@@ -13,6 +13,39 @@ import InfoBox from 'shared/components/InfoBox';
...
+* The Palette agent is now able to automatically upgrade to newer versions.
+
+## Packs
+
+* CNI Packs:
+  * Calico CNI 3.25.0
+* CSI Packs:
+  * EBS CSI 1.16.0
+  * Vsphere CSI 2.7.0
```
Comment (ritawatson) [inline content/docs/00-release-notes.md:43]:
```suggestion
  * vSphere CSI 2.7.0
```

**Review by ritawatson — APPROVED** [review]
A couple minor fixes. Otherwise LGTM

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1185#" title="Author self-approved">karl-cardenas-coding</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1185#pullrequestreview-1342531845" title="Approved">ritawatson</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/release-3-3/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: content/docs/00-release-notes.md [hunk 4]
```diff
@@ -13,6 +13,39 @@ import InfoBox from 'shared/components/InfoBox';
...
+## Edge
+
+### Features:
+
+* Edge now supports the ability to load images from an external OCI registry.
+* The Edge Installer can now include preloaded content bundles containing packages and artifacts. This is useful for scenarios where you work with limited internet bandwidth or want to optimize the installation process.
+* Users can create a custom Edge Installer that contains content bundles, operating systems, and user data.
+* Support for creating an Edge Installer ISO based on Virtual Machine Disks (VMDK) is now available.
+* Support for generating random UUID values for the Edge host is now available. This addresses the issue of some devices having the same Universal Unique Identifier (UUID) due to identical device identifiers.
+* The Palette agent is now able to automatically upgrade to newer versions.
```
Comment (karl-cardenas-coding) [inline content/docs/00-release-notes.md:35]:
```suggestion
* The Palette Edge Host agent can now upgrade to newer versions automatically.
```

**Pull request merged**